packages/cedar/src/Chart.ts

```diff
@@ -114,7 +110,7 @@ export class Chart {
 
     if (definition) {
       // set the definition
-      this.definition(clone(definition))
+      this.definition(definition)
```

Comment:
technically this is a breaking change, but since `chart.defintion(newDef)` didn't clone, I figure we could make it consistent